code_snippets/api-metric-metadata-get.py

```diff
@@ -0,0 +1,12 @@
+from datadog import initialize, api
+
+options = {
+    'api_key': '9775a026f1ca7d1c6c5af9d94d9595a4',
+    'app_key': '87ce4a24b5553d2e482ea8a8500e71b8ad4554ff'
+}
+
+initialize(**options)
+
+metric = 'system.cpu.idle'
+
+print api.Metadata.get(metric_name=metric)
```

Comment:
Found out that this is caused by the order of the decorators on the API method. I have a PR out to fix this in dogweb so that valid json is returned for auth errors.
https://github.com/DataDog/dogweb/pull/17238